src/uitpas/search/detail/UiTPasAdvantageDescription.js

```diff
@@ -124,6 +135,32 @@ export default class UiTPasAdvantageDescription extends React.Component {
         );
     }
 
+    isOnlineExchangable() {
+        const cashInType = this.props.advantage.cashInType;
+        return cashInType ? cashInType.includes(CashInType.ONLINE) : false;
+    }
+
+    renderIsOnlineExchangable() {
+        if (this.isOnlineExchangable()) {
+            return <div className="online-exchangable">online om te ruilen</div>
+        }
+
+        return null;
+    }
+
+    onlineExchangeUrl() {
+        console.log('env', process.env.REACT_APP_UITPAS_URL);
+        return `${process.env.REACT_APP_UITPAS_URL}/omruilen/${this.props.advantage.id}`;
```

Comment:
Can't you use the existing `PUBLIC_URL` here?

Reply:
No, it's something else

Reply:
Please clarify next time what is then next time like @brampauwelyn did here https://github.com/cultuurnet/uitpas-voordelen-search/pull/31#discussion_r664423832